dev.Dockerfile

```diff
@@ -71,7 +71,9 @@ WORKDIR /usr/local/src/watcher
 RUN git clone https://github.com/e-dant/watcher . && \
 	cmake -S . -B build -DCMAKE_BUILD_TYPE=Release && \
 	cmake --build build/ && \
-	cmake --install build
+	cmake --install build && \
+	cp build/libwatcher-c.so /usr/local/lib/libwatcher-c.so && \
+	ldconfig
```

Comment:
following this comment https://github.com/dunglas/frankenphp/issues/1156#issuecomment-2471737748